content/tutorials/get-started/look-up-transaction-results.md

```diff
@@ -0,0 +1,435 @@
+
+
+## 1. Get Transaction Status
+
+Knowing whether a transaction succeeded or failed is a two-part question:
+
+1. Was the transaction included in a validated ledger?
+2. If so, what changes to the ledger state occurred as a result?
+
+To know whether a transaction was included in a validated ledger, you usually need access to all the ledgers it could possibly be in. The simplest, most foolproof way to do this is to look up the transaction on a [full history server](ledger-history.html#full-history). Use the [tx method][], [account_tx method][], or other response from `rippled`. Look for `"validated": true` to indicate that this response uses a ledger version that has been validated by consensus.
+
+- If the result does not have `"validated": true`, then the result may be tentative and you must wait for the ledger to be validated to know if the transaction's outcome is final.
+- If the result does not contain the transaction in question, or returns the error `txnNotFound`, then the transaction is not in any ledger that the server has in its available history. This may or may not mean that the transaction failed, depending on whether the transaction could be in a validated ledger version that the server does not have and whether it could be included in a future validated ledger. You can constrain the range of ledgers a transaction can be in by knowing:
+    - The earliest ledger the transaction could be in, which is the **first ledger to be validated _after_ the transaction was first submitted**.
+    - The last ledger the transaction could be in, which is defined by the transaction's `LastLedgerSequence` field.
+
+The following example shows a successful transaction, as returned by the [tx method][], which is in a validated ledger version. The order of the fields in the JSON response has been rearranged, with some parts omitted, to make it easier to understand:
+
+```json
+{
+  "TransactionType": "AccountSet",
+  "Account": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+  "Sequence": 376,
+  "hash": "017DED8F5E20F0335C6F56E3D5EE7EF5F7E83FB81D2904072E665EEA69402567",
+
+  ... (omitted) ...
+
+  "meta": {
+    "AffectedNodes": [
+      ... (omitted) ...
+    ],
+    "TransactionResult": "tesSUCCESS"
+  },
+  "ledger_index": 46447423,
+  "validated": true
+}
+```
+
+This example shows an **[AccountSet transaction][]** sent by the [account](accounts.html) with address **rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn**, using [Sequence number][] 376. The transaction's [identifying hash][] is `017DED8F5E20F0335C6F56E3D5EE7EF5F7E83FB81D2904072E665EEA69402567` and its [result](transaction-results.html) is `tesSUCCESS`. The transaction was included in ledger version 46447423, which has been validated, so these results are final.
```

Comment:
Just curious -- why are AccountSet transaction and the address value bolded?

Reply:
Originally I did it to highlight specific details that aren't strictly code samples (especially the address). But, given the way the rest of the sentence (and indeed, the whole document) turned out, there would have to be so much bold as to lose all meaning. So, on second look, I've decided the simplest solution is to remove the bold here. Thanks for your attention to detail!